blocks/edit/da-library/helpers/helpers.js

```diff
@@ -232,3 +232,32 @@ export function delay(ms) {
   // eslint-disable-next-line no-promise-executor-return
   return new Promise((resolve) => setTimeout(resolve, ms));
 }
+
+export const getMetadata = (el) => [...el.childNodes].reduce((rdx, row) => {
```

Comment:
This function can be imported from here (double check to verify it's exactly the same):
https://github.com/adobe/da-live/blob/main/blocks/edit/utils/helpers.js#L420